src/background.js

```diff
@@ -384,8 +386,9 @@ async function updateBrowserActionIcon (tab) {
       });
       browser.browserAction.setBadgeText({tabId: tab.id, text: " "});
     }
-  } else {
-    browser.browserAction.setPopup({tabId: tab.id, popup: "./panel2.html"});
+  } else { 
+    browser.storage.local.set({"CURRENT_PANEL": "no-trackers"});
```

Comment:
This is what's setting the panel back to the "no-trackers" state when it's re-opened. Filed https://github.com/mozilla/contain-facebook/issues/270